pyproject.toml

```diff
@@ -114,6 +114,7 @@ dependencies = [
   "types-tqdm",
   "types-requests",
   "types-protobuf",
+  "openai<1.0.0",
```

Comment:
question: why doesn't this inherit from the default?

Reply:
non blocking but @axiomofjoy any ideas?

Reply:
I'm not clear on why this dependency needed to be added. Our type checks were failing without it.